plugins/modules/redhat_subscription.py

```diff
@@ -24,6 +24,11 @@
       I(server_proxy_hostname), I(server_proxy_port), I(server_proxy_user) and
       I(server_proxy_password) are no longer taken from the C(/etc/rhsm/rhsm.conf)
       config file and default to None.
+    - Since 6.5.0, credentials (i.e. I(username) and I(password), or I(activationkey), or I(token))
+      are needed only in case the the system is not registered, or I(force_register) is specified;
+      this makes it possible to use the module to tweak an already registered system, e.g. attaching
+      pools to it (using I(pool) I(pool_ids)), and modifying the syspurpose attributes
```

Comment:
Just wondering, should this be formatted differently?
```suggestion
      pools to it (using I(pool) I(pool_ids)), and modifying the C(syspurpose) attributes
```

Reply:
here "syspurpose" is referring to the "syspurpose attributes" of a system, rather than the "syspurpose" parameter of the module; the actual attribute is mentioned right after this in brackets:

> and modifying the syspurpose attributes (using I(syspurpose))

Reply:
`C(...)` is not for parameters anyway, but for values/program names/whatever names that are not regular text.